golang/cosmos/x/swingset/keeper/swing_store_exports_handler.go

```diff
@@ -157,44 +157,76 @@ type swingStoreRestoreExportAction struct {
 	Args        [1]swingStoreImportOptions `json:"args"`
 }
 
-// SwingStoreExportModeCurrent represents the minimal set of artifacts needed
-// to operate a node.
-const SwingStoreExportModeCurrent = "current"
+// SwingStoreNoneArtifactLevel mean that no artifacts are part of the
```

Comment:
```suggestion
// SwingStoreNoneArtifactLevel means that no artifacts are part of the
```
And a nit: I think we should probably give these constants left-to-right names for supporting purpose identification, e.g. SwingStore{None,Operational,Replay,Archival,Debug}ArtifactLevel → SwingStoreArtifactLevel{None,Operational,Replay,Archival,Debug} and SwingStore{Skip,RepairMetadata,All}ExportDataMode → SwingStoreExportDataMode{Skip,RepairMetadata,Full}, and also group them visually like
```go
const (
	…ArtifactLevel constants…
)

const (
	…ExportDataMode constants…
)
```